adhocracy-plus/assets/scss/components/_project_tile_carousel.scss

```diff
@@ -9,14 +9,15 @@
         :focus,
         :hover {
             outline: none;
-
+            /* stylelint-disable max-nesting-depth */
             .project-tile-carousel__dot {
```

Comment:
@Rineee I decided to just disable the linter for the over nested styling

Reply:
Cool, thanks!